tests/unit/BitbucketContext/Application/CommandHandler/Inspection/SendViolationsAsCommentsTest.php

```diff
@@ -87,4 +87,21 @@ public function testAnInspectionWithNoReportTriggersNothing()
 
         $this->handler->handle($command);
     }
+
+    public function testAnInspectionForAFlightModeRepositoryTriggersNothing()
+    {
+        $inspection = $this->createMock(PullRequestInspection::class);
+        $repository = $this->createMock(Repository::class);
+
+        $inspection->method('hasReport')->willReturn(true);
+        $inspection->method('getRepository')->willReturn($repository);
+
+        $repository->method('isFlightModeEnabled')->willReturn(true);
+
+        $this->reporter->expects($this->never())->method('report');
+
+        $command = new Command\Inspection\SendViolationsAsComments($inspection);
```

Comment:
Parameter #1 $inspection of class Regis\BitbucketContext\Application\Command\Inspection\SendViolationsAsComments constructor expects Regis\BitbucketContext\Domain\Entity\PullRequestInspection, PHPUnit_Framework_MockObject_MockObject given.